extensions/vertx-web/runtime/src/main/java/io/quarkus/vertx/web/RouteFilter.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.vertx.web;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import io.vertx.ext.web.RoutingContext;
+
+/**
+ * This annotation is used to define a "filter", i.e. a route called on every HTTP request.
+ * <p>
+ * The target business method must return {@code void} and accept exactly one argument of type {@link RoutingContext}.
```

Comment:
Might be useful to write a few words on the default priority. WDYT?

Reply:
Hm, I don't know what to say. `FilterBuildItem` has some pre-defined system priorities but I can't reference them in the runtime module. I chose `10` so that it's called after these system filters...

Reply:
Maybe just mentioned that the default priority is 10.

Reply:
Ok, will do.